builtin/k8s/task_test.go

```diff
@@ -0,0 +1,41 @@
+package k8s
+
+import (
+	"fmt"
+	"github.com/hashicorp/hcl/v2"
+	"github.com/hashicorp/hcl/v2/hclsyntax"
+	"github.com/hashicorp/waypoint-plugin-sdk/component"
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+func TestStartTask(t *testing.T) {
```

Comment:
This is a great test! I think a comment explaining how to use it would be helpful here. It looks like it won't run at all if the tester doesn't have an env var HCL_FILE set, and a file to go along with it.

Reply:
Added the file